examples/Atlas/Atlas.m

```diff
@@ -79,6 +82,13 @@
         end
       end
 
+      % Add a force on a specified link if we want!
+      if ~isempty(options.external_force)
+        obj.external_force = 1;
```

Comment:
Changing this to store the actual linkID of the force being applied; it was being used as a boolean here, but I don't think there's a reason not to keep track of the linkID proper. Thanks!